Triphouse-project/index.html

```diff
@@ -8,12 +8,72 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="./style.css" />
-    <title>Document</title>
+    <title>Triphouse</title>
 </head>
-
 <body>
     <header>
+        <div class="header__main">
+            <div class="header__menu">
```

Comment:
![2021-04-19_22h24_57](https://user-images.githubusercontent.com/19935417/115292060-2bdf8a00-a15e-11eb-9ef1-42b360cf15f6.png)
https://t.me/c/1421457090/1292
Investigate example, your variant of implementation isn't good.

Reply:
Fixed underline